source/parameters/bend.md

```diff
@@ -85,8 +85,7 @@ Zero `e1_rect` and `e2_rect` gives a rectangular magnet shape.
 - **edge_int1, edge_int2**
 The field integral for the entrance pole face `edge_int1` is given by
   ```{math}
-  \text{edge_int1} = \int_{pole} \!\! ds \frac{B_y(s) (B_{y0} - B_y(s))}
-  {2 , B_{y0}^2}
+  \text{edge_int1} = \int_{pole} \!\! ds \frac{B_y(s) (B_{y0} - B_y(s))}{2 B_{y0}^2}
```

Comment:
@DavidSagan 

I'm bringing the components of the fraction onto the same line, in case this breaks the LaTeX build. 

Was the `,` between `2` and `B_{y0}^2` intentional? I wonder if this may contribute to breaking the build.

Reply:
The comma should not have been there. It accidentally got left in in the translation from latex.